toqito/channel_ops/natural_representation.py

```diff
@@ -0,0 +1,46 @@
+"""Kraus operators to natural representation."""
+
+import numpy as np
+
+from toqito.matrix_ops import tensor
+
+
+def natural_representation(kraus_ops):
```

Comment:
We should include type-hints for `kraus_ops` along with type-hints for the return type.